hardhat.config.ts

```diff
@@ -0,0 +1,44 @@
+import '@nomiclabs/hardhat-ethers';
+import dotenv from 'dotenv';
+import { HardhatUserConfig } from 'hardhat/config';
+
+dotenv.config({ path: './.env' });
+const ALCHEMY_ID = typeof process !== 'undefined' ? process.env.ALCHEMY_ID : '';
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+
+const config: HardhatUserConfig = {
+  defaultNetwork: 'tenderly',
+  solidity: {
+    version: '0.8.19',
+  },
+  networks: {
+    tenderly: {
+      chainId: 10,
+      url: process.env.RPC_URL || '',
+      gas: 20000000,
+    },
+    local: {
+      chainId: 31337,
+      url: process.env.CUSTOM_LOCAL_RPC || '',
```

Comment:
This should be just `LOCAL_RPC`. Feel free to rename those things separately though, in another PR.